src/routes/dictionaries/+page.svelte

```diff
@@ -30,16 +30,32 @@
   let createDictionaryError: string | null = null;
   let creatingDictionary: boolean = false;
   let files: FileList;
+  let file: File;
+  let fileInput: HTMLInputElement;
+  let isSequenceAdaptation: boolean = false;
+  let sequenceAdaptationName: string;
 
   $: hasCreatePermission = featurePermissions.commandDictionary.canCreate(data.user);
-  $: createButtonDisabled = !files;
+  $: createButtonDisabled = !files || (isSequenceAdaptation && sequenceAdaptationName === '');
```

Comment:
Need to check for existence of a file within `files` instead of just the existence of the potentially empty `files` object. Otherwise, the create button will not be properly disabled in all cases.